pyproject.toml

```
@@ -44,3 +46,9 @@ profile = "black"
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.mypy]
+plugins = ["mypy_django_plugin.main"]
+
+[tool.django-stubs]
+django_settings_module = "myproject.settings"
```

Add a new line at the end of the file.
```suggestion
django_settings_module = "myproject.settings"

```